embedded_postgres.go

```diff
@@ -51,6 +52,7 @@ func newDatabaseWithConfig(config Config) *EmbeddedPostgres {
 
 // Start will try to start the configured Postgres process returning an error when there were any problems with invocation.
 // If any error occurs Start will try to also Stop the Postgres process in order to not leave any sub-process running.
+//nolint:funlen
```

Comment:
Would be great to avoid this but I'll tidy up in a later commit. The below may help a little!